### ISSUE TITLE
Bug:  Navigation Drawer is missing overlay + focus trap

### ISSUE THREAD

**Issue by synnaf** [Frojd/Frojd-Jewl#4]
There should be an overlay for Navigation Drawer when expanded, as when doing a modal popup. 
Tabbing should be focused inside the menu, not reach links or other elements outside. 

https://github.com/Frojd/Frojd-Jewl/blob/f22b4ee1d7b15e2ba7c0761777ebd8756dcd90d9/component-library/app/components/NavigationDrawer/NavigationDrawer.module.scss#L33
